add info re: MR comments with multiple GitLab groups (#1705)
semgrep/semgrep-docs: docs/semgrep-appsec-platform/gitlab-mr-comments.md

1. In GitLab, go to [<i class="fas fa-external-link fa-xs"></i> Profile > Access Tokens](https://gitlab.com/-/profile/personal_access_tokens), and then add a token with `api` scope.
1. Copy the token created in the previous step.

> Once you have a GitLab PAT, you can provide it to Semgrep through [Semgrep AppSec Platform's (SAP) **<i class="fa-solid fa-gear"></i> Settings > Source Code Managers** tab](/deployment/connect-scm#gitlab-self-managed-plans). The SAP is a central location that makes it easy for your teams to access and rotate the token if necessary. This is the recommended method, though there are two other options for providing your PAT to Semgrep:

Reviewer: ```suggestion
Once you have a GitLab PAT, you can provide it to Semgrep through [Semgrep AppSec Platform's**<i class="fa-solid fa-gear"></i> Settings > Source Code Managers** tab](/deployment/connect-scm#gitlab-self-managed-plans). This tab is a central location that makes it easy for your teams to access and rotate the token if necessary. This is the recommended method, though there are two other options for providing your PAT to Semgrep:
```
Extremely minor, but I'd suggest we not acronymize SAP where possible :)

> 2. In the [Network Broker configuration file](/semgrep-ci/network-broker#configure-semgrep-network-broker). If the Gitlab user requires a PAT, you can set a dummy token and assign the PAT in the Network Broker configuration file.

Reviewer: ```suggestion
- In the [Network Broker configuration file](/semgrep-ci/network-broker#configure-semgrep-network-broker): If the Gitlab user requires a PAT, you can set a dummy token and assign the PAT in the Network Broker configuration file.
```

> 3. In the [CI job's configuration file](/semgrep-ci/sample-ci-configs#sample-gitlab-cicd-configuration-snippet). This is helpful if you're using Gitlab's Project Access Tokens, which are generated on a per-project basis.

Reviewer: ```suggestion
3. In the [CI job's configuration file](/semgrep-ci/sample-ci-configs#sample-gitlab-cicd-configuration-snippet): This is helpful if you're using Gitlab's Project Access Tokens, which are generated on a per-project basis.
```

For more configuration options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs#gitlab-cicd).

:::info MR comments with multiple GitLab groups
If you're using Semgrep with multiple GitLab groups, ensure that you've completed one of the following steps to ensure that you see MR comments for repositories associated with all of your groups:

> 1. Create a Semgrep account connection with each GitLab group by signing in to your Semgrep account, navigating to **Settings > Source code managers**, and adding an entry for each group.

Reviewer 2: Since gitlab supports the concept of sub-groups, perhaps we might want to mention that you only need to add a single entry for the parent group when you want to onboard ALL subgroups of a parent group.

> 2. Add the `GITLAB_TOKEN` environment variable to your CI configuration. You can set `GITLAB_TOKEN` to the GitLab PAT that grants access to all of your GitLab groups (recommended), or you can use a different GitLab PAT for each of your GitLab groups.

Reviewer 2: ```suggestion
2. Add the `GITLAB_TOKEN` environment variable to your CI configuration. You can set `GITLAB_TOKEN` to the GitLab PAT that grants access to all of your GitLab groups (if possible), or you can use a different GitLab PAT for each of your GitLab groups.
```
Using a single GitLab PAT for all GitLab groups is not always possible (depending on the GL license)
